ott/tools/sinkhorn_divergence.py

```diff
@@ -148,6 +148,7 @@ def _sinkhorn_divergence(
 def segment_sinkhorn_divergence(
     x: jnp.ndarray,
     y: jnp.ndarray,
+    padding_vector: Optional[Callable[..., jnp.ndarray]] = None,
```

Comment:
I do not think padding_vector needs to be specified.

By default, the cost is Euclidean, the padding vector should be 0.

If a cost function (available in kwargs for instance) is passed (which can be tested below), then the padding vector should be computed from there, and then passed to the segmenting function (as suggested above for `bar_problems`)

Reply:
what you are proposing is what I had implemented initially as a solution.

however, I thought that it might be confusing to the user that he has to pass the cost in ``segment_sinkhorn_divergence``, so I thought that I would go for ``padding_vector`` which might be more clear to the user why it is needed